### PR TITLE
El Capitan support for pl2303

### PR DESCRIPTION
**Pull request by Ibuprofen**
- El capitan support

### CODE REVIEW

File: Casks/pl2303.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  url "http://prolificusa.com/files/md_PL2303_MacOSX_10.6-10.10_v#{version}.zip"
+  url "http://www.prolific.com.tw/UserFiles/files/PL2303_MacOSX_#{version.gsub('.','_')}.zip"
   name 'Prolific USB-Serial Cable driver'
   homepage 'http://www.prolificusa.com'
   license :closed
 
   pkg "PL2303_MacOSX_v#{version}.pkg"
 
-  depends_on :macos => [:snow_leopard, :lion, :mountain_lion, :mavericks, :yosemite]
+  depends_on :macos => [:snow_leopard, :lion, :mountain_lion, :mavericks, :yosemite, :el_capitan]
```
Comment (vitorgalvao) [inline Casks/pl2303.rb]: Remove this, if every version is now supported.

File: Casks/pl2303.rb [hunk 2]
```diff
@@ -1,17 +1,17 @@
 cask :v1 => 'pl2303' do
-  version '1.5.1'
-  sha256 '101058b71da6a6bd038b05abef0e556bd3203fcd04f35d545fd711c5d34d46c2'
+  version '1.6.0_20151022'
+  sha256 'ed0d7965c57e6d855e1909e0af7228372bedb0ae47247a00c5eea519b5dde6bb'
 
-  url "http://prolificusa.com/files/md_PL2303_MacOSX_10.6-10.10_v#{version}.zip"
+  url "http://www.prolific.com.tw/UserFiles/files/PL2303_MacOSX_#{version.gsub('.','_')}.zip"
   name 'Prolific USB-Serial Cable driver'
   homepage 'http://www.prolificusa.com'
```
Comment (vitorgalvao) [inline Casks/pl2303.rb]: Would it make sense to change this to `https://www.prolific.com/`?

File: Casks/pl2303.rb (hunk 1 above)
Comment (Ibuprofen) [inline Casks/pl2303.rb]: It is safe to assume that future versions will work?

File: Casks/pl2303.rb (hunk 2 above)
Comment (Ibuprofen) [inline Casks/pl2303.rb]: Nerp: `curl: (7) Failed to connect to www.prolific.com.tw port 443: Connection refused`

Comment (Ibuprofen) [inline Casks/pl2303.rb]: I've looked around but don't see where or if the file is located a that domain.

File: Casks/pl2303.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/pl2303.rb]: Yes.

File: Casks/pl2303.rb [hunk 3]
```diff
@@ -1,17 +1,17 @@
 cask :v1 => 'pl2303' do
-  version '1.5.1'
-  sha256 '101058b71da6a6bd038b05abef0e556bd3203fcd04f35d545fd711c5d34d46c2'
+  version '1.6.0_20151022'
+  sha256 'ed0d7965c57e6d855e1909e0af7228372bedb0ae47247a00c5eea519b5dde6bb'
 
-  url "http://prolificusa.com/files/md_PL2303_MacOSX_10.6-10.10_v#{version}.zip"
+  url "http://www.prolific.com.tw/UserFiles/files/PL2303_MacOSX_#{version.gsub('.','_')}.zip"
```
Comment (vitorgalvao) [inline Casks/pl2303.rb]: > I've looked around but don't see where or if the file is located at that domain.

In that case, where have you found this download url?

Comment (Ibuprofen) [inline Casks/pl2303.rb]: via: http://www.prolific.com.tw/US/ShowProduct.aspx?p_id=229&pcid=41

File: Casks/pl2303.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/pl2303.rb]: > via: http://www.prolific.com.tw/US/ShowProduct.aspx?p_id=229&pcid=41

In that case, would it make sense to change this to `https://www.prolific.com.tw/`? That is what I meant before.

Comment (vitorgalvao) [inline Casks/pl2303.rb]: > via: http://www.prolific.com.tw/US/ShowProduct.aspx?p_id=229&pcid=41

In that case,w ould it make sense to change this to `http://www.prolific.com.tw/`? That is what I meant before.

**Comment by Ibuprofen** [comment]
How do those changes look? Thanks.

**Comment by vitorgalvao** [comment]
Looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by Ibuprofen** [comment]
All squishy! Thanks.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It was merged directly as a57f035a32039e8328cf3df92a89f10f114ef63f to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**